foodsaving/utils/frontend_urls.py

```diff
@@ -46,7 +46,7 @@ def invite_url(invitation):
 
 
 def user_delete_url(code):
-    return '{hostname}/#/user/delete?code={code}'.format(
+    return '{hostname}/#/delete-user?code={code}'.format(
```

Comment:
`user/delete` is in conflict with `user/:id`. To resolve this, I can think of 3 options:
1. Change `user/delete` to some other URL (this is what I've done)
2. Change `user/:id` to some other URL
3. Add `user/delete` to the vue-router before `user/:id`. This would be the best option in my opinion, but it seems to be difficult considering the implementation of `router.js` (which loads main-layout URLs before splash-layout URLs).

Reply:
1. seems easiest to me
2. would break existing URLs
3. is a bit tricky, maybe it would make sense to rethink our route implementation. For example, `groups/create` and `groups/:id` both exist and mostly work fine, but I also had a case where it failed in a subtle way. Would not recommend right now.